chain/cosmos/08-wasm-types/consensus_state.go

```diff
@@ -0,0 +1,19 @@
+package types
+
+import (
+	"github.com/cosmos/ibc-go/v7/modules/core/exported"
+)
+
+var _ exported.ConsensusState = (*ConsensusState)(nil)
+
+func (m ConsensusState) ClientType() string {
+	return ""
```

Comment:
Is the client type `""`?

Reply:
Same as above, the implementation of these 08-wasm interfaces does not matter, we just want the codecs.